setup.py

```diff
@@ -1,27 +1,29 @@
 # -*- coding: utf-8 -*-
-"""`sphinx_rtd_theme` lives on `Github`_.
-
-.. _github: https://github.com/readthedocs/sphinx_rtd_theme
-
-"""
 
+import distutils.cmd
```

Comment:
Picky: I prefer this below os and subprocess

Reply:
I used isort for this

Reply:
In that case, don't worry about it, I don't have a strong preference.